nidaqmx_examples/nidaqmx_warnings.py

```diff
@@ -2,24 +2,25 @@
 import warnings
 
 import nidaqmx
+from nidaqmx.error_codes import DAQmxWarnings
 
 # By default warnings are shown.
 with nidaqmx.Task() as task:
-    task.ao_channels.add_ao_voltage_chan("cDAQ1Mod3/ao0")
+    task.ao_channels.add_ao_voltage_chan("Dev1/ao0")
```

Comment:
We should add a `.ini` for the devices used by examples.

Also, we should make sure that adding these devices doesn't break the other tests. Currently, it seems to do that:

```
FAILED tests/legacy/test_read_write.py::TestPowerRead::test_power_n_chan_n_samp[3832426340985976644-output_enables1] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
FAILED tests/legacy/test_read_write.py::TestPowerRead::test_power_n_chan_n_samp[5669956416450651358-output_enables2] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
FAILED tests/legacy/test_read_write.py::TestPowerRead::test_power_n_chan_n_samp[421582776643551488-output_enables3] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
FAILED tests/legacy/test_read_write.py::TestPowerRead::test_mixed_chans_with_power[2999364054301950491] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
FAILED tests/legacy/test_stream_power_readers.py::TestPowerMultiChannelReader::test_power_n_chan_1_samp[2693308682444374927-output_enables0] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
FAILED tests/legacy/test_stream_power_readers.py::TestPowerMultiChannelReader::test_power_n_chan_1_samp[281155321069432379-output_enables1] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
FAILED tests/legacy/test_stream_power_readers.py::TestPowerMultiChannelReader::test_power_n_chan_1_samp[7873677367780628324-output_enables2] - nidaqmx.errors.DaqError: Multidevice tasks cannot use the on-demand sample timing type. Configure timing to synchro...
```